x-pack/plugins/lens/public/xy_visualization/xy_suggestions.ts

```diff
@@ -117,8 +117,6 @@ function flipSeriesType(seriesType: SeriesType) {
       return 'bar_stacked';
     case 'bar':
       return 'bar_horizontal';
-    case 'bar_horizontal_stacked':
-      return 'bar_stacked';
```

Comment:
This doesn't look right to me, why was this removed?

Reply:
It's a duplicate. At line 116 there's exactly the same case.